.dockerignore

```diff
@@ -19,6 +19,4 @@ run
 .env.local
```

Comment:
can you add those files, they are not useful in production images: 

```
Dockerfile
LICENSE
renovate.json
tests
```

Reply:
For some reason, github is not detecting the change in the code